io.github.electron.geforcenow.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally metainfo, screenshots, desktop file and icon should be maintained in the upstream repo and not here.